examples/facerec_from_webcam.py

```diff
@@ -39,19 +40,22 @@
     # Find all the faces and face enqcodings in the frame of video
     face_locations = face_recognition.face_locations(rgb_frame)
     face_encodings = face_recognition.face_encodings(rgb_frame, face_locations)
-
+    
     # Loop through each face in this frame of video
     for (top, right, bottom, left), face_encoding in zip(face_locations, face_encodings):
         # See if the face is a match for the known face(s)
         matches = face_recognition.compare_faces(known_face_encodings, face_encoding)
-
+        
         name = "Unknown"
-
+        
+        # Calculate face distance
+        face_distance = face_recognition.face_distance(known_face_encodings, face_encoding)
         # If a match was found in known_face_encodings, just use the first one.
         if True in matches:
             first_match_index = matches.index(True)
-            name = known_face_names[first_match_index]
-
+            # Sort nearest distance
+            name = known_face_names[np.argsort(face_distance)[0]]
```

Comment:
`name = known_face_names[np.argmin(face_distance)]`

That's faster and more explicit :)